tests/test_entry.py

```diff
@@ -5,9 +5,12 @@
 
 from __future__ import division, print_function, unicode_literals
 
+from os import path
 from os import stat
+from contextlib import closing
+import tarfile
 
-from libarchive import memory_reader, memory_writer
+from libarchive import memory_reader, memory_writer, file_reader
```

Comment:
The two `os` imports should be only one, and everything should be kept in alphabetical order.